spec/lucky/cookies/cookie_jar_spec.cr

```diff
@@ -70,14 +70,26 @@ describe Lucky::CookieJar do
     jar.get?(:name).should be_nil
   end
 
-  it "raises helpful error if trying to read unencrypted values" do
+  it "returns nil if fails to decrypt value" do
     jar = Lucky::CookieJar.empty_jar
 
     jar.set_raw(:name, "Jane")
 
-    expect_raises Exception, "cookies.get_raw(:name).value" do
-      jar.get?(:name)
-    end
+    jar.get?(:name).should be_nil
+  end
+
+  it "parses encrypted cookies as expected" do
```

Comment:
Hopefully this spec will help in the future if we have to change cookies again

Reply:
I think this makes sense, but it's also going to be difficult to maintain this test if it needs to change for some reason because of the opaque cookie value. I know it's redundant because the crypto code exists in the other file, but does it make sense to put a few lines of explicit crypto in here which builds the cookie value instead of the raw base64 encoded value?